module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -182,12 +182,18 @@ namespace sys
             systemDeinit();
         }
 
+        // We disable all uninitialized devices
+        deviceManager->DisableAllDevices();
+
         // Power off request (pending)
         PowerOff();
 
         //  End of scheduler and back to the main and poweroff
         EndScheduler();
 
+        powerManager.reset();
+        cpuStatistics.reset();
+        deviceManager.reset();
```

Comment:
it should never call this. If needed please call it before EndScheduler.

Reply:
done